package-lock.json

```diff
@@ -30,7 +30,7 @@
         "@bcrs-shared-components/interfaces": "1.1.5",
         "@bcrs-shared-components/jurisdiction": "1.0.1",
         "@bcrs-shared-components/limited-restoration-panel": "1.0.5",
-        "@bcrs-shared-components/mixins": "1.1.34",
+        "@bcrs-shared-components/mixins": "1.1.38",
```

Comment:
Name Request Mixin now emits an error's text (instead of displaying a browser alert).